themes/default/layouts/partials/registry/package/header.html

```diff
@@ -24,6 +24,38 @@
 
     <div class="lg:flex mb-2">
         <h1 class="text-4xl no-anchor">{{ with .GetPage $packageHome }}{{ .Title }}{{ end }}</h1>
+
+        {{ with (index $.Site.Data.registry.packages ($packageName)) }}
+            <div class="flex-grow justify-end">
+                {{ if (eq (.publisher) "Pulumi") }}
+                    <div class="flex lg:justify-end mt-2.5">
+                        <pulumi-tooltip>
+                            <div class="h-10 flex py-2 px-6 bg-blue-100 rounded-xl items-center">
```

Comment:
Also a somewhat minorish thing, but with the blue background, rounded corners, and cursor on hover, this looks a lot like a clickable button.

Reply:
@anita-trimbur any thoughts on this?

Reply:
That's a fair point, especially on account of the hover. That said, we do need some extra emphasis and separation from other text on the page to maintain the proper information hierarchy. We already use an underline on this page for selected tabs. Further reducing the opacity of that bg color could help mitigate the button association. @susanev, what do you think?

Reply:
@anita-trimbur sure, id be good with reducing the opacity a bit. we actually have a similar problem in docs right now so id also be good shipping as is and revisiting the use of our primary color.